docs/development/maintenance.md

```diff
@@ -13,104 +15,124 @@ modules of ILIAS.
+
+For the context of ILIAS, we define **four** different authorities:
+
+1. **Authority to Sign of Code Changes**
+2. **Authority to Sign of Conceptual Changes**
+3. **Authority to Sign of Testcase Changes**
+4. **Authority to (De-)Assign Authorities**
+
+Each of these authorities can be held by a different set of people. This means
+that the social organisation of different groups working on different parts of
+the Code of ILIAS can be different.
+Right now ILIAS knows a few different social structures for the maintenance of
+units in the code of ILIAS:
+
+* In the **"Classic Model"** all authorities are concentrated in one person and
+this person works mostly alone.
+* In the **Coordinator Model** all authorities are concentrated in one or more
+people and they work together with other developers in the community to improve
+the code.
+* In the **"Test and Assessment Model"** the authorities **to Sign of Conceptual
+Changes**, **to Sign of Testcase Changes**, and **to (De-)Assign Authorities**
+lies with one person and the **Authority to Sign of Code Changes** with two
+others.
+
+More will surely emerge as the optimal solution for each unit is found.
+
+# Responsibilites
+Independently of the social organisation for each unit of code the following
+responsibilites need to be assumed:
+
+* All people holding an authority must agree to coordinate the development
+of their unit of code with the product manager and with the people maintaining
+other units of code.
+* One of the people holding either the **Authority to Sign of Code Changes** or
+the **Authority to Sign of Conceptual Changes** gets assigned related bugs
+automatically by the [Issue-Tracker](https://mantis.ilias.de). S/he is responsible
+to make sure all issues receive a response within the defined time frame and are
+either fixed in a timely manner or postponed/closed with a solid explanation.
+* The people holding the **Authority to Sign of Code Changes** are responsible
+for Pull Requests to their component and get assigned related Pull Requests
+by the Technical Board according to the [Rules for Maintainers and Coordinators
+assigned to PRs](https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/docs/development/contributing.md#rules-for-maintainers-assigned-to-prs).
+* The person/people holding the **Authority to (De-)Assign Authorities**
+coordinate assignments of authorities with the Product Manager and the Technical
+Board, who hold a vetoing power over these decisions.
+
+# Additional Rules and Guidelines
+* Although the first decision on new features or feature removals in a unit of
+code lie with the person/people holding the **Authority to Sign of Conceptual
+Changes** the final decisions are made by them together with the product
+manager during the Jour Fixe meetings after an open discussion.
+* If nobody holds the **Authority to (De-)Assign Authorities** for a code unit
+it authomatically defaults to the Technical Board.
```

Comment:
„authomatically“ → „automatically“ (oder besser: Wort streichen).

Ich würde vorschlagen:
```markdown
* When nobody holds the **Authority to (De-)Assign Authorities** for a code unit,
that authority is wielded by the Technical Board.
```

Jemand mit besseren Englischkenntnissen muss nur etwas dazu sagen, ob „to wield“ der richtige Begriff (hier für „ausüben“ gedacht) ist, und ihn ggf. ersetzen.

Reply:
Ich nehme mal einfach das falsch "authomatically" raus.

Reply:
Dankeschön.